src/natcap/invest/urban_mental_health.py

```diff
@@ -0,0 +1,1335 @@
+"""InVEST Urban Mental Health model."""
+import logging
+import os
+import shutil
+import tempfile
+
+import numpy
+import pandas
+import pygeoprocessing
+import pygeoprocessing.kernels
+from osgeo import gdal
+from osgeo import ogr, osr
+
+from . import gettext
+from . import spec
+from . import validation
+from .unit_registry import u
+
+LOGGER = logging.getLogger(__name__)
+
+SCENARIO_OPTIONS = [
```

Comment:
I noticed this is only used in the MODEL_SPEC and in Validation. Will this be useful to have as a constant with later options? Otherwise it might read a little better if the options were in the spec itself. But then of course they'd be duplicated for the validation function...